modules/lib/private-endpoint/main.bicep

```diff
@@ -0,0 +1,65 @@
+@description('Location for all Private Endpoint(s).')
+param location string
+
+@description('Array of Private Endpoint(s) to create.')
+param privateEndpoints array = []
```

Comment:
It would be better if we can use User-defined data types which is now available with bicep 0.17.1